phoebe/frontend/io.py

```diff
@@ -465,8 +465,16 @@ def load_legacy(filename, add_compute_legacy=True, add_compute_phoebe=True,
         raise TypeError("filename must be string or file object, got {}".format(type(filename)))
 
     # load the phoebe file
-    params = np.loadtxt(filename, dtype='str', delimiter='=',
-                        converters={0: lambda s: s.strip(), 1: lambda s: s.strip()})
+    params = np.genfromtxt(
+        filename,
+        dtype='str',
+        delimiter='=',
+        converters={0: lambda s: s.strip().decode() if isinstance(s, bytes) else s.strip(),
+                    1: lambda s: s.strip().decode() if isinstance(s, bytes) else s.strip()},
+        comments='#',       
+        filling_values='',   
+        autostrip=True
```

Comment:
does `autostrip` remove the need for `.strip` in the converter?

Reply:
Yes, you're right. I just tested three combinations:
- without `autostrip = True`
- without `strip()` in converters
- without converters

All of them passed the tests for Python 3.9 and 3.10. I'll leave it up to you to decide which should I keep. The most convenient is probably the one without `converters` and with `autostrip=True`, but the one closest to the original is without `autostrip = True`.